frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -223,7 +225,11 @@ class DatasetImportView extends React.PureComponent<Props, State> {
         datasetClone.dataSource.status = dataset.dataSource.status;
         dataset = (datasetClone: APIDataset);
       }
-      this.props.form.setFieldsValue({
+      const form = this.formRef.current;
+      if (!form) {
+        return;
```

Comment:
This is the "template" @MichaelBuessemeyer introduced everywhere `form` is accessed in a React class component. I agree that it would be better to throw an error here, instead of failing silently :+1:

Reply:
Ok, cool. I changed it in this particular line, since this seemed like it could cause a weird state theoretically. For the other places, I left it as is. It's probably not important in the end, anyway :pray: